algebra/src/main/scala/endpoints/MethodsAlg.scala

```diff
@@ -0,0 +1,16 @@
+package endpoints
+
+
+trait MethodsAlg {
```

Comment:
The current naming convention is to use singular nouns for algebra interfaces: `MethodAlg`.